sources/vega/commands/v1/validator_commands.proto

```diff
@@ -82,6 +84,8 @@ message KeyRotateSubmission {
   uint32 key_number = 1;
   // Target block at which the key rotation will take effect on
   uint64 target_block = 2;
-  // Hash of the public key to rotate too
+  // The new public key to rotate to
   string new_pub_key = 3;
+  // Hash of currently used public key
+  string current_pub_key_hash = 4;
```

Comment:
We may need validation in the command folder for this new field (even for the whole command if not don before btw)